docs/settings.rst

```diff
@@ -202,6 +202,9 @@ Options that change ticket updates
 
   **Default:** ``HELPDESK_STAFF_ONLY_TICKET_CC = False``
 
+- **HELPDESK_SHOW_CUSTOM_FIELDS_FOLLOW_UP_LIST** Show configured custom fields in the follow-up form.
```

Comment:
I would have add an example to show that fields don't need to be prefixed by `custom_`